src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -820,5 +821,13 @@ public class SimpleEvents {
 					.since("2.9.0")
 					.requiredPlugins("Paper");
 		}
+
+		Skript.registerEvent("Broadcast", SimpleEvent.class, BroadcastMessageEvent.class, "broadcast")
+			.description("Called when a message is broadcasted.")
+			.examples(
+				"on broadcast:",
+					"\tset broadcast-message to \"&c[BROADCAST] %broadcasted message%\""
+			)
+			.since("INSERT VERSION");
```

Comment:
Is all these broadcast message event stuff supposed to be in this pull request?

Reply:
Yeah it's used for testing that the async JUnit events work, should I create a seperate PR for it?